Osiris/Hacks/Misc.cpp

```diff
@@ -412,6 +426,24 @@ void Misc::drawBombTimer() noexcept
     }
 }
 
+void Misc::drawFov() noexcept
+{
+    auto local = localPlayer.get();
+    if (config->misc.drawFov)
+    {
+        if (!local || !local->isAlive()) return;
+        int weaponId = getWeaponIndex(localPlayer->getActiveWeapon()->itemDefinitionIndex2());
+        if (!config->aimbot[weaponId].enabled) weaponId = 0;
+        if (!config->aimbot[weaponId].enabled) return;
+        auto screenSize = interfaces->surface->getScreenSize();
+        if (config->aimbot[weaponId].silent) interfaces->surface->setDrawColor(255, 10, 10, 255);
+        else interfaces->surface->setDrawColor(10, 255, 10, 255);
+        float radius = std::tan(degreesToRadians(config->aimbot[weaponId].fov) / 2.f) / std::tan(degreesToRadians(local->isScoped() ? local->fov() : (105 + config->visuals.fov)) / 2.f) * screenSize.first;
```

Comment:
float radius = std::tan(degreesToRadians(config->aimbot[weaponIndex].fov) / 2.f) / std::tan(degreesToRadians(globals::currentFOV) / 2.f) * width;

globals::currentFOV = setup->fov;

With your method drawfov wont be accurate with different FOV.

Reply:
It remains inaccurate.